.github/scripts/upload_benchmark_results.py

```diff
@@ -172,7 +172,8 @@ def upload_to_dynamodb(
     """
     Copied from upload stats script
     """
-    info(f"Writing {len(docs)} documents to DynamoDB {dynamodb_table}")
+    msg = f"Writing {len(docs)} documents to DynamoDB {dynamodb_table}"
```

Comment:
This comes from https://google.github.io/styleguide/pyguide.html#3101-logging, I'm lazy and just move the message to a variable to silent the warning.